pkg/koordlet/metriccache/metric_cache.go

```diff
@@ -181,6 +201,22 @@ func (m *metricCache) GetPodResourceMetric(podUID *string, param *QueryParam) Po
 		return result
 	}
 
+	GPUMetricsArray := make([][]gpuResourceMetric, 0)
```

Comment:
why do we initialize `[][]gpuResourceMetric` instead of `[]gpuResourceMetric`?

Reply:
support for multiple device.